package.json

```diff
@@ -44,6 +44,7 @@
     "rimraf": "^6.0.1"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-free": "^6.6.0",
```

Comment:
We shouldn't be using font awesome. If we need iconography, let's use [USWDS's icons](https://designsystem.digital.gov/components/icon/).

Reply:
What andrew said ^^